### ISSUE TITLE
Resource Timings add important attributes

### ISSUE THREAD

**Issue by ceckoslab** [basicrum/backoffice#33]
I started working on #31 but I notice that it is not easy to identify which resource are blocking and which not. I see a need that we keep more data bout page resources in order to solve this task.

I will be good to have extra data Resource Timings data in our database because this will open possibility for developing more features like:
 - Calculating the total size of JS on page.
 - Number of blocking resources.
 - And more report related features.

**Comment by ceckoslab** [comment]
My **draft** plan is to add more columns in **resource_timings** table:

- initiator_type
- rel
- async
- defer
- included_in_head
- encoded_body_size
- decoded_body_size
- transfer_size

Also the values of those new columns/properties will be used to define resource uniqueness because sometimes a resource may have same url but e.g. has different size or an attribute like defer may change.

I also should check the case when an resource is an image because for image we keep information like:
- height
- width
- y position 
- x position
- natural_height
- natural_width

**Comment by ceckoslab** [comment]
Change in the plan. I worked about a week in order to design a nice data structure in order to persist all this information in MySql DB. I got some good results but started experiencing Memory leaks during imports.

I decided that at this stage this does not bring much value to the project but actually it makes it hard to develop and understand.

I decided to read all the needed data from DB field *beacons.beacon* and keep things simple. This also will lead to faster and simple import process.
